flink-connector/flink-connector-gcp-pubsub/src/test/java/com/google/pubsub/flink/PubSubSourceTest.java

```diff
@@ -58,6 +58,27 @@ public void build_invalidSchema() throws Exception {
     assertThrows(IllegalStateException.class, builder::build);
   }
 
+  @Test
+  public void build_invalidMaxOutstandingMessagesCount() throws Exception {
```

Comment:
do we want to throw an exception on negative values? (here and below)

Reply:
Good idea! I added throwing an exception on negative or zero values. This mimics the behavior of the underlying library that uses these settings. Surfacing the exception earlier, when building `PubSubSource`, will make debugging failing Flink jobs much easier.